docs: how to run  godwoken-kicker on Windows11 with WSL

ShookLyngs opened the pull request.

keroro520 commented on docs/kicker-start-on-windows-11.md:
```diff
@@ -0,0 +1,97 @@
+- Windows -  22000.708
+- Docker - 20.10.16
+- Docker Compose - 2.6.0
+- WSL 2 OS - Ubuntu 22.04 LTS
```
It would be very helpful for some Windows users that note Godwoken-Kicker can not run on the Windows systems and why.

// As this note is so important, I mark this message as a request change.

keroro520 commented on docs/kicker-start-on-windows-11.md:
```diff
@@ -0,0 +1,97 @@
+- Clone kicker into Windows’ File System - Good for Windows based developers, but has a few adjustment works to do, otherwise the kicker tool is non-usable
+- Clone kicker into WSL 2 subsystem - Good for the kicker tool to execute since maintainers are mostly using macOS or Linux, Windows based developers will need to adapt for a while
+
+## Clone Godwoken-Kicker in Windows’ File System
+
+1. Go to a folder where you store projects, for example `D:/projects`
+2. Open a terminal, and then clone the kicker tool into the projects folder
+3. Go into the kicker tool’s root folder (for example `D:/projects/godwoken-kicker`), make sure all files in the folder are in `LF` format, if not then: 
+   1. run `git config --local core.autocrlf input` to change the git configuration for the project
+   2. run `git reset --hard head` to rebuild files so now all files are well-formatted
+4. In root folder of the kicker tool, there’s a file called `kicker`, change its filename to `kicker.sh` so Windows Terminal can execute it
+5. Open a terminal, run command `./kicker.sh start` to start building a Godwoken Devnet
+
+More friendly tips:
+
+- You might want to go to Windows Terminal’s setting page, then go to Profiles → Defaults → Advanced, find `Profile termination behavior` and set it to `Never close automatically`, this will prevent the kicker tool from closing itself automatically, after its task ends
```
The best practice is "cloning in WSL 2 subsystem", right? If so, I believe introducing this only would be sufficient.

"Clone Godwoken-Kicker in Windows’ File System" is Windows-domain knowledge, or rather, personal knowledge. I suggest removing this part or moving it to the end of this document as a part of "known issues".

Flouse replied:
suggest removing this part too.

The best practice with one choice is better for this case.

keroro520 commented on docs/kicker-start-on-windows-11.md:
```diff
@@ -0,0 +1,97 @@
+
+## Clone Godwoken-Kicker in WSL 2 Subsystem
+
+In this path, the most important task is to connect to Git in the subsystem, and for that we need to set up an SSH key. So we basically have 2 options, pick one you like:
+
+- Use `ssh-keygen` to generate SSH key in subsystem, then add it on GitHub
+    
+    [How to Use SSH with GitHub (Instead of HTTPS) on Windows WSL](https://simplernerd.com/git-ssh-keys/)
+    
+- Copy SSH key from Windows to Linux subsystem
+    
+    [Sharing SSH keys between Windows and WSL 2](https://devblogs.microsoft.com/commandline/sharing-ssh-keys-between-windows-and-wsl-2/)
```
Usually, as this is an open-source repo, we recommend users clone via HTTPS, without SSH key or password required.

keroro520 commented on docs/kicker-start-on-windows-11.md:
```diff
@@ -0,0 +1,97 @@
+
+After that, we can begin to run the kicker tool in our subsystem:
+
+1. Go to the home folder of the current user, for example `/home/<username>`
+2. Create a folder to store your projects, for example `~/projects`
+3. Go into projects folder, and then clone the kicker tool into the folder
```
Missed the important part: `git clone https://github.com/RetricSu/godwoken-kicker`.

ShookLyngs commented:
@keroro520 The article is updated:
1. Added a section about why windows-based developer might need the article
2. Deleted section about working on Windows' File System
3. Deleted section about setting a SSH key
